src/cpp/rtps/xmlparser/XMLParser.cpp

```diff
@@ -1848,7 +1848,7 @@ XMLP_ret XMLParser::fillDataNode(
                 return XMLP_ret::XML_ERROR;
             }
         }
-        else if (strcmp(name, THROUGHPUT_CONT) == 0)
+        else if (strcmp(name, THROUGHPUT_CONT) == 0)    // TODO (Ricardo) Deprecated. Remove in the future.
```

Comment:
Should we add a logWarning about deprecation?

Reply:
Done